launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -64,6 +64,10 @@
   /** Configuration key for the number of executor CPU cores. */
   public static final String EXECUTOR_CORES = "spark.executor.cores";
 
+  public static final String PYSPARK_DRIVER_PYTHON = "spark.pyspark.driver.python";
```

Comment:
Yeah. I know it sucks but I think it's worth it to avoid adding more things in the public API, given it's just two strings. You could add a small test in `SparkLauncherSuite` to make sure everything is kept consistent (since that test is in the same package as the launcher code).